native-image-support/src/main/resources/META-INF/native-image/com.google.cloud/google-cloud-core/native-image.properties

```diff
@@ -0,0 +1,14 @@
+Args = --allow-incomplete-classpath \
+--enable-url-protocols=https,http \
+--initialize-at-build-time=org.conscrypt \
```

Comment:
What's calling this in?

Reply:
These arguments are used for the `native-image` compiler which is used to build the native image. It includes all common the settings that are required for native image compilation to work.

For example:
As part of some optimizations, Native Image tends to initialize certain classes at build-time (when the native image is being built). However, some classes/values cannot be initialized at build-time, which results in errors that look like this: `Classes that should be initialized at run time got initialized during image building` so we need to specify which ones need to be initialized at run-time. The `initialize-at-build-time` and `-initialize-at-run-time` args are used for this purpose.